flow-typed/file-system.js

```diff
@@ -0,0 +1,4 @@
+declare module 'file-system' {
```

Comment:
Definitely don't want this module.   Instead use https://www.npmjs.com/package/mz:
* https://github.com/solana-labs/example-tictactoe/blob/master/package.json#L46
* https://github.com/solana-labs/example-tictactoe/blob/master/flow-typed/npm/mz_vx.x.x.js